packages/eslint-config-eventbrite-react/rules/react.js

```diff
@@ -85,41 +106,84 @@ module.exports = {
         // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-pascal-case.md
         'react/jsx-pascal-case': 'error',
 
-        // Enforce that there's a space before the closing bracket of self-closing JSX elements
+        // Forbid spaces after the opening bracket
+        // Forbid spaces before the closing bracket
+        // Enforce spaces before the closing bracket of self-closing elements
+        // Forbid spaces between the angle bracket and slash of JSX closing or self-closing elements
         // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-space-before-closing.md
-        'react/jsx-space-before-closing': 'error',
+        'react/jsx-tag-spacing': ['error', {
+            afterOpening: 'never',
+            beforeClosing: 'never',
+            beforeSelfClosing: 'always',
+            closingSlash: 'never',
+        }],
 
         // Enforce multi-line JSX is wrapped in parentheses
         // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-wrap-multilines.md
         'react/jsx-wrap-multilines': 'error',
 
+        // Prevent usage of `this.state` inside `setState` calls
+        // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-access-state-in-setstate.md
+        'react/no-access-state-in-setstate': 'error',
```

Comment:
:tada: YUSS!!

Reply:
This is my favorite one. I'm always commenting on this in reviews